public/css/drag-drop-style.css

```
@@ -0,0 +1,523 @@
+  overflow: hidden;
+  background-color: #343434;
+}
+
+#progressBar {
+  height: 100%;
+  width: 10%;
+  background-color: orange;
+  transition: width 1.5s ease-in-out;
+}
+
+@keyframes zoomIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "zoomIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)